src/main/java/core/basesyntax/TriangleArea.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface TriangleArea {
+    default double getTriangleArea(double firstLeg, double secondLeg) {
+        return firstLeg * secondLeg * 0.5;
```

Comment:
All magic numbers in your code should be constants. The number '0.5' is considered a magic number. Define it as a constant with a clear name to improve code readability and maintainability.